src/app/store/Cart/Cart.dispatcher.js

```diff
@@ -122,15 +122,15 @@ export class CartDispatcher {
             if (type_id === 'configurable') {
                 let configurableVariantIndex = 0;
 
-                const { product: variant } = variants.find(
-                    ({ product }, index) => {
-                        const { sku: productSku } = product;
+                const selectedVariant = variants
+                    .filter(({ product }) => product !== null)
+                    .find((variant, id) => {
```

Comment:
Instead of filtering before finding (those are two array iterations) it is preferred to just enhance find function, so it does not return true not break on empty (null) product.